src/contracts/BPool.sol

```diff
@@ -6,14 +6,18 @@ import {BMath} from './BMath.sol';
 import {BToken, IERC20} from './BToken.sol';
 import {IBPool} from 'interfaces/IBPool.sol';
 
+/**
+ * @title BPool
+ * @notice Pool contract that allows for entering and exiting positions and swapping of tokens.
```

Comment:
this is a weird description for an AMM contract, i'd say that holds the tokens, allows to swap, and add and remove liquidity, or sth like that, i'd remove the term "position"